src/picosvg/svg_types.py

```diff
@@ -285,15 +286,20 @@ def callback(curr_xy, cmd, args, prev_xy, prev_cmd, prev_args)
                 if y_coord_idxs:
                     new_y += new_cmd_args[y_coord_idxs[-1]]
 
+                # z takes the pen to the return_pos
+                if new_cmd.lower() == "z":
+                    new_x, new_y = return_pos
+
                 prev_pos = copy.copy(curr_pos)
```

Comment:
same here. we don't need copy

Reply:
heh I have mutable points on the brain! Fixing.